openshift/template.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: 3scale-admin-gateway
+metadata:
+  name: 3scale-admin-gateway
+  annotations:
+    description: 3scale-admin-gateway
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      service: 3scale-admin-gateway
+    name: 3scale-admin-gateway
+  spec:
+    selector:
+      service: 3scale-admin-gateway
+    template:
+      metadata:
+        labels:
+          service: 3scale-admin-gateway
+      spec:
+        containers:
+        - env:
+          - name: F8A_DEBUG
+            value: "true"
+          - name: DEPLOYMENT_PREFIX
+            valueFrom:
+              configMapKeyRef:
+                name: bayesian-config
+                key: deployment-prefix
+          - name: BAYESIAN_FETCH_PUBLIC_KEY
```

Comment:
Do we need to talk to keycloak from this service (?) I don't know, I am asking :)